docs/important/relevant_functionalities_de.md

```diff
@@ -0,0 +1,1250 @@
+
+Dogus, die anderen Dogus gegenüber Zugangskonten bereitstellen können, nennt man auch Service-Account-Produzent. Dogus, die ein Zugangskonto bei einem anderen Dogu benötigen, nennt man Service-Account-Konsument.
+
+### Service Accounts produzieren 
+
+Service Accounts werden nicht von einem Konsumer-Dogu selbst angefragt, weil ein Dogu sich nicht um diese Form der Orchestrierung kümmern sollte. Dies ist stattdessen die Aufgabe von einem Client wie `cesapp` oder `k8s-dogu-operator` während der Installation des Service-Account-Konsument-Dogus. Das `ExposedCommand` hierfür lautet `service-account-create`. 
+
+Ähnlich hierzu sieht die Löschung von Service Accounts aus, denn ebenfalls kümmert sich ein Client wie `cesapp` oder `k8s-dogu-operator` während der Deinstallation eines Dogus darum, dass dessen angeforderte Service Accounts wieder gelöscht werden. Das `ExposedCommand` hierfür lautet `service-account-remove`.
+
+Damit ein Dogu als Service-Account-Produzent auftreten kann, muss es in seiner `dogu.json` zwei ausführbare Dateien exponieren:
+
+```json
+{
+  ...,
+  "ExposedCommands": [
+    {
+      "Name": "service-account-create",
+      "Description": "Creates a new service account",
+      "Command": "/create-sa.sh"
+    },
+    {
+      "Name": "service-account-remove",
+      "Description": "Removes a service account",
+      "Command": "/remove-sa.sh"
+    }
+  ],
+  ...,
+}
+```
+
+Die Begriffe `service-account-create` und `service-account-remove` sind geschützte Eigenschaften und werden von Clients, die die `dogu.json` interpretieren als Befehle zum Erstellen und Löschen von Service Accounts benutzt.
+
+In den beiden folgenden Abschnitten wird auf die Erzeugung und Löschung von Service Accounts eingegangen. Weitere Information über ExposedCommands befinden sich im [ExposedCommands](../core/compendium_de.md#exposedcommands)-Abschnitt des Kompendiums.
+
+#### Service Account anlegen
+
+Das `service-account-create`-Skript verwaltet das Erstellen eines neuen Service Accounts. Es wird immer ausgeführt, wenn ein neuer Konsument installiert wird. Während des Aufrufs erhält das `service-account-create`-Skript jedenfalls den Namen des Konsumenten-Dogus als erstes Argument. Ein zweites Argument zur Feineinstellung des Service Accounts ist optional und hängt von der Unterstützung des Produzenten-Dogus ab.
+
+Ein `service-account-create`-Skript besteht aus drei Schritten:
+
+1. Erstellen der Account Informationen
+2. Anlegen eines Accounts für unsere Software mit den Informationen
+3. Schreibe die Accountinformationen verschlüsselt in den Dogu-spezifischen Konfigurationsbereich des Verbrauchers.
```

Comment:
```suggestion
3. Verschlüsseltes Schreiben der Accountinformationen in den Dogu-spezifischen Konfigurationsbereich des Verbrauchers
```

Reply:
Done. I rephrased the whole sentence  to emphasize the actual task at hand